### PR TITLE
Setup project skeleton for v2

### PR DESCRIPTION
**Pull request by gsvr**

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,4 @@
-language: python
-python:
-  - 2.6
-  - 2.7
-script:
-    - ./run-tests.sh
+language: node_js
+node_js:
+  - "0.11"
+  - "0.10"
```
Comment (imsickofmaps) [inline .travis.yml]: Line break

File: Gruntfile.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                options: {
+                    atBegin: true
+                }
+            }
+        },
+
+        concat: {
+            options: {
+                banner: [
+                    '// WARNING: This is a generated file.',
```
Comment (imsickofmaps) [inline Gruntfile.js:69]: :heart_eyes:

File: src/changemanagement.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+var vumigo = require('vumigo_v02');
```
Comment (imsickofmaps) [inline src/changemanagement.js:1]: These (and all other app src files requires) should be inside your `go.module = {`

File: src/changemanagement.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+var ChoiceState = vumigo.states.ChoiceState;
+var EndState = vumigo.states.EndState;
+var Choice = vumigo.states.Choice;
+
+
+go.cm = {
+    // Registration of Head Teacher States
```
Comment (imsickofmaps) [inline src/changemanagement.js]: Did this comment get updated in future versions? If not, update now?

**Comment by imsickofmaps** [comment]
@gsvr just a couple of tweaks

File: src/changemanagement.js (hunk 3 above)
Comment (gsvr) [inline src/changemanagement.js:1]: Will fix in #117 to avoid merging complications.

**Comment by imsickofmaps** [comment]
:+1: land away

**Pull request merged**